python/cudf/cudf/_libxx/column.pyx

```diff
@@ -0,0 +1,167 @@
+import numpy as np
+import cython
+import rmm
+
+from libc.stdint cimport uintptr_t
+from libcpp.pair cimport pair
+from libcpp cimport bool
+
+from cudf._libxx.lib cimport *
+from cudf.core.buffer import Buffer
+from libc.stdlib cimport malloc, free
+
+from cudf.utils.dtypes import is_categorical_dtype
+
+
+np_to_cudf_types = {np.dtype('int8'): INT8,
+                    np.dtype('int16'): INT16,
+                    np.dtype('int32'): INT32,
+                    np.dtype('int64'): INT64,
+                    np.dtype('float32'): FLOAT32,
+                    np.dtype('float64'): FLOAT64,
+                    np.dtype("datetime64[D]"): TIMESTAMP_DAYS,
```

Comment:
TIMESTAMP_DAYS is a 32 bit INT underneath, `datetime64[D]` will return an `itemsize` of 8 which doesn't align with this. I think we should omit this for now.

Reply:
Done